example/position.py

```diff
@@ -0,0 +1,24 @@
+import curses
+import pick
+
+def main(stdscr):
+    stdscr.addstr("hello world?\n")
+    stdscr.get_wch()
+
+    y, x = stdscr.getyx()
+    
+    title = "Please choose your favorite programming language: "
+    options = ["Java", "JavaScript", "Python", "PHP", "C++", "Erlang", "Haskell"]
+    option, index = pick.pick(
+        options,
+        title,
+        indicator="=>",
+        default_index=2,
+        screen=stdscr,
+        position=pick.Position(y, 0)  # comment this to demonstrate the issue it solves
```

Comment:
Using `yx` instead of `xy` is a little bit rare, so in the example specify them with the key would make it more clear.

```suggestion
        position=pick.Position(y=y, x=0)  # comment this to demonstrate the issue it solves
```